### PR TITLE
Bugfix/proxybackend form close validation

### PR DESCRIPTION
**Pull request by jykae**
Closes https://github.com/apinf/platform/issues/1935

### CODE REVIEW

**Comment by marla-singer** [comment]
@jykae Reviewing

**Comment by marla-singer** [comment]
@jykae It shows sAlert if frontend prefix isn't unique. But it shows as well if i change Proxy after that

**Steps**
1. Select Proxy №1 and fill no unique value in **/proxy_frontend_prefix/** field
2. Watch on sAlert
3. Select another Proxy, don't change value in **/proxy_frontend_prefix/** field (and this value is unique for another Proxy)

**Expected**
Result will be saved
**Found**
Message about "no unique path"

If my steps are not clear, let me know

**Comment by marla-singer** [comment]
@jykae Another question: does it need that fronted prefix must be unique for total proxies? Not in scope of one proxy?

**Comment by marla-singer** [comment]
Well, looks like uniqueness must be totally.

File: proxy_backends/server/methods.js [hunk 1]
```diff
@@ -34,4 +34,12 @@ Meteor.methods({
       }
     );
   },
+  uniqueFrontendPrefix (proxyBackend) {
+    const p = ProxyBackends.findOne({ 'apiUmbrella.url_matches.frontend_prefix': proxyBackend.apiUmbrella.url_matches[0].frontend_prefix });
```
Comment (marla-singer) [inline proxy_backends/server/methods.js]: Please rename this variable to sensible value

**Comment by jykae** [comment]
```
@jykae Another question: does it need that fronted prefix must be unique for total proxies? Not in scope of one proxy?
```
Good note. We might have logical error on this one, as we previously had just one proxy. Checking only proxyBackends for current proxy would be enough.

**Comment by marla-singer** [comment]
@jykae Let's pass this checking on a new issue?

**Comment by jykae** [comment]
@marla-singer well, if you think so. I think I could check that also as part of this PR using "proxyId" on the selector as well, but it will go for tomorrow.

**Comment by bajiat** [comment]
@marla-singer @brylie Can we merge / finalize this PR or should I move this to the next sprint?

**Comment by marla-singer** [comment]
@bajiat I'm going to finalize it today

**Comment by bajiat** [comment]
Cool!

**Comment by marla-singer** [comment]
@bajiat Fix the little remarks, merge this branch and close issue

**Pull request merged**